src/base/projects/Header.svelte

```diff
@@ -134,29 +134,29 @@
     on:click={() => toggleContent(ProjectContent.History, true)}>
     <strong>{tree.stats.commits}</strong> commit(s)
   </div>
-  {#if project.issues > 0}
+  {#if project.issues}
     <div
       class="stat issue-count clickable widget"
+      class:not-allowed={project.issues === 0}
+      class:clickable={project.issues > 0}
       class:active={content == ProjectContent.Issues}
-      on:click={() => toggleContent(ProjectContent.Issues, false)}>
+      on:click={() => {
+        if (project.issues && project.issues > 0) toggleContent(ProjectContent.Issues, false); }
```

Comment:
Seems like these checks should be redundant no? Due to line 137, 141

Reply:
Yeah I wanted to be extra explicit, but by hiding or at least not adding a `clickable` class, we can obtain the same result